tests/components/samsungtv/test_config_flow.py

```diff
@@ -1292,7 +1292,7 @@ async def test_autodetect_legacy(hass: HomeAssistant) -> None:
     )
     assert result["type"] is FlowResultType.CREATE_ENTRY
     assert result["data"][CONF_METHOD] == "legacy"
-    assert result["data"][CONF_NAME] == "fake_name"
+    assert result["data"][CONF_NAME] is None
```

Comment:
Why are we testing is None?

Reply:
Only because other fields are tested for None as well.

PR updated

Reply:
I think conf_name needs a thorough review, as it seems to be passed and used by the legacy bridge

Reply:
If `name` was set previously, then we still might want to respect this.